Allow compiler to be specified with CC. Rename nonstandard C_FLAGS and
L_FLAGS variables to the standard CFLAGS and LDFLAGS respectively.
--- Makefile.orig	2002-11-01 10:46:05.000000000 -0600
+++ Makefile	2021-02-12 23:40:02.000000000 -0600
@@ -1,32 +1,33 @@
 all:	f77_diagram
 
-C_FLAGS = -O2 -Wall
-L_FLAGS =
+CC ?= gcc
+CFLAGS = -O2 -Wall
+LDFLAGS =
 
 f77_diagram:	block.o do_loop.o f77_diagram.o if.o label.o links.o parse_fortran.o
-		gcc $(L_FLAGS) -o f77_diagram block.o do_loop.o f77_diagram.o if.o label.o \
+		$(CC) $(LDFLAGS) -o f77_diagram block.o do_loop.o f77_diagram.o if.o label.o \
 		    	          links.o parse_fortran.o
 		
 block.o:	     block.c block.h
-		     gcc $(C_FLAGS) -c block.c
+		$(CC) $(CFLAGS) -c block.c
 
 do_loop.o:	     do_loop.c do_loop.h
-		     gcc $(C_FLAGS) -c do_loop.c
+		$(CC) $(CFLAGS) -c do_loop.c
 
 f77_diagram.o:	f77_diagram.c
-		gcc $(C_FLAGS) -c f77_diagram.c
+		$(CC) $(CFLAGS) -c f77_diagram.c
 
 if.o:		if.c if.h
-		gcc $(C_FLAGS) -c if.c
+		$(CC) $(CFLAGS) -c if.c
 
 label.o:	label.c label.h
-		gcc $(C_FLAGS) -c label.c
+		$(CC) $(CFLAGS) -c label.c
 
 links.o:	links.c links.h
-		gcc $(C_FLAGS) -c links.c
+		$(CC) $(CFLAGS) -c links.c
 
 parse_fortran.o:    parse_fortran.c parse_fortran.h
-		    gcc $(C_FLAGS) -c parse_fortran.c
+		$(CC) $(CFLAGS) -c parse_fortran.c
 
 out.dot:	./f77_diagram
 		./f77_diagram -o out.dot plot.for
